mailet/api/src/main/java/org/apache/mailet/Mail.java

```diff
@@ -233,7 +236,14 @@ default boolean hasSender() {
      */
     @Deprecated
     Serializable getAttribute(String name);
-    
+
+    /**
+     * Returns the attribute corresponding to an attribute name
+     *
+     * @param name the attribute name
+     * @return the attribute
```

Comment:
IMO `@return` and `@param` are not adding value here. IMO either we make it more useful or we can drop it - it repeat type and variable name.